index.html

```diff
@@ -4,484 +4,528 @@
-                    alt="location"
-                  />
-                  <input
-                    class="location-input"
-                    type="text"
-                    placeholder="Location"
-                    name="text"
-                    value=""
-                  />
-                </form>
+        <!-- Navigation menu -->
+        <nav class="nav-menu">
+          <ul class="nav-menu__list">
+            <li class="nav-menu__list-item"><a href="#home">Home</a></li>
+            <li class="nav-menu__list-item">
+              <a href="#categories">Categories</a>
+            </li>
+            <li class="nav-menu__list-item"><a href="#service">Service</a></li>
+            <li class="nav-menu__list-item"><a href="#jobs">Jobs</a></li>
+            <button class="button nav__button" href="#">Post a vacancy</button>
+          </ul>
+        </nav>
+        <nav class="menu nav-menu">
+          <!-- Navigation list items -->
+          <ul class="menu__list">
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#home">Home</a>
+            </li>
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#categories">Categories</a>
+            </li>
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#service">Service</a>
+            </li>
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#jobs">Jobs</a>
+            </li>
+          </ul>
+        </nav>
 
-                <!-- Search button -->
-                <div class="hero-section__button">
-                  <a class="buttons buttons__search-button" href="#">
-                    Search
-                  </a>
-                </div>
-              </div>
-            </div>
+        <!-- Navigation button -->
+        <button class="button button__cta" href="#">Post a vacancy</button>
+      </div>
+    </header>
 
-            <!-- Right wrapper -->
-            <div class="right-wrapper">
-              <img
-                class="right-wrapper__img"
-                src="./src/img/satisfied-lovely-woman-holds-modern-cell-phone-1.png"
-                alt="satisfied-lovely-woman-holds-modern-cell-phone"
-              />
+    <main>
+      <!-- Hero section -->
+      <section class="hero">
+        <!-- Hero section wrapper -->
 
-              <!-- Big ellipse -->
-              <img
-                class="right-wrapper__ellipse-big"
-                src="./src/img/ellipse-big.svg"
-                alt="big ellipse"
-              />
-              <!-- Small ellipse -->
-              <img
-                class="right-wrapper__ellipse-small"
-                src="./src/img/ellipse-small.svg"
-                alt="small ellipse"
-              />
+        <!-- Info -->
+        <div class="info">
+          <h1 class="info__title title">Find & Apply For Your Dream Job</h1>
```

Comment:
```suggestion
          <h1 class="title">Find & Apply For Your Dream Job</h1>
```

1. I think it's a block, not an element. Elements can't be used outside their blocks, but blocks may be used anywhere. So you may use this title anywhere on your website. In this case the proper naming will be `info-title`.
Please, read about [BEM](https://ru.bem.info/methodology/quick-start/) one more time.

2. I suggest using universal class names like `title`, `subtitle` etc, and add them extra properties by writing CSS selectors this way: `.info .title {}`. So you won't need extra class names to add margins etc for a single usage.